android/expoview/src/main/java/versioned/host/exp/exponent/modules/api/GoogleModule.java

```diff
@@ -117,6 +117,13 @@ public void run() {
     });
   }
 
+  @ReactMethod
+  public void signOut() {
```

Comment:
Make this an async function -- even though there is no value returned, returning from a function (i.e. resolving a promise in the async case) also conveys that the function finished running.